### PR TITLE
tests: Move Windows unit tests to CircleCI

### PR DESCRIPTION
**Pull request by sidharthachatterjee**
Because Azure pipelines are too slow

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -107,5 +107,8 @@
   },
   "workspaces": [
     "packages/*"
-  ]
+  ],
+  "resolutions": {
+    "write-file-atomic": "2.4.1"
```
Comment (wardpeet) [inline package.json]: Any chance you can elaborate on this change? :)

Comment (sidharthachatterjee) [reply package.json]: Oh right! Sorry! Jest has a known issue that causes a race condition on Windows and breaks some tests. This is the recommended fix in the issue that tracks this bug (https://github.com/facebook/jest/issues/4444)

File: package.json [hunk 2]
```diff
@@ -107,5 +107,8 @@
...
+  }
```
Comment (wardpeet) [inline package.json]: We could add something like this to keep track on the error. Maybe creating an issue in gatsby and tagging it with upstream is a good place too 😛 
```suggestion
  },
  "comments": {
    "resolutions": {
      "write-file-atomic": "Jest bug on windows: https://github.com/facebook/jest/issues/4444"
    }
  }
```

Comment (sidharthachatterjee) [reply package.json]: Documented in https://github.com/gatsbyjs/gatsby/issues/16885

**Pull request merged**